homework/Contributor.js

```diff
@@ -14,6 +14,11 @@ class Contributor {
    */
   render(container) {
     // TODO: replace the next line with your code.
-    Util.createAndAppend('pre', container, JSON.stringify(this.contributor, null, 2));
+    const contributorName = Util.createAndAppend('li', container);
+    contributorName.innerHTML += `<a target ="_blank" href= ${
```

Comment:
Is there any reason why you didn't use `createAndAppend` method from Utils class here to create this innerHTML?